test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven-tm/before/src/main/java/foo/bar/TransferManagerS3.java

```diff
@@ -48,6 +51,18 @@ void download(TransferManager tm, String bucket, String key) {
 
     void upload(TransferManager tm, String bucket, String key) {
         tm.upload(bucket, key, file);
+
+        File file = new File("file1.txt");
+        PutObjectRequest requestWithFile = new PutObjectRequest(bucket, key, file);
+        tm.upload(requestWithFile);
+
+        InputStream inputStream = new ByteArrayInputStream(("HelloWorld").getBytes());
+        PutObjectRequest requestWithInputStream = new PutObjectRequest(bucket, key, "location");
+        requestWithInputStream.setInputStream(inputStream);
+        tm.upload(requestWithInputStream);
+
+        PutObjectRequest requestWithoutPayload = new PutObjectRequest(bucket, key, "location");
```

Comment:
Does this actually upload empty content?

Reply:
Yup 0 bytes object will be uploaded